e2e-tests/IIoTPlatform-E2E-Tests/TestModels/PublishNodesRequestApiModel.cs

```diff
@@ -0,0 +1,43 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace IIoTPlatform_E2E_Tests.TestModels {
+    using System.Collections.Generic;
+    using System.ComponentModel.DataAnnotations;
+    using System.Runtime.Serialization;
+
+    /// <summary>
+    /// A monitored and published item
+    /// </summary>
+    public class PublishNodesRequestApiModel {
```

Comment:
do we really need to duplicate the API classes? can't we just take a dependency on the publisher api project?

Reply:
The nuget package is not updated and the publisher api project is in a different solution